test/script.clj

```diff
@@ -34,6 +34,79 @@
        (with-out-str (aws/doc s3 :ListBuckets))
        "Returns a list of all buckets")))
 
+(require '[pod.babashka.aws.credentials :as creds])
+
+(defmacro with-system-properties [props & body]
+  `(let [props# (System/getProperties)]
+     (try
+       (doseq [[k# v#] ~props]
+         (System/setProperty k# v#))
+       ~@body
+       (finally
+         (System/setProperties props#)))))
+
+(defn create-temp-dir [prefix affix]
```

Comment:
There's a method for that in java nio:)

https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory(java.nio.file.Path,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...)

Reply:
197b4c2133ef7213461daf0e2b854f939b3cc507